.rustfmt.toml

```diff
@@ -0,0 +1,13 @@
+use_small_heuristics = "Default"
+force_explicit_abi = true
+hard_tabs = false
+max_width = 100
+merge_derives = true
+newline_style = "Auto"
+remove_nested_parens = true
+reorder_imports = true
+reorder_modules = true
+tab_spaces = 4
+use_field_init_shorthand = true
+use_try_shorthand = true
```

Comment:
`false` is the default for this one, which I changed to `true`, because that's what all of our code looks like.
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#use_try_shorthand

Reply:
[Based on the docs](http://doc.rust-lang.org/1.32.0/core/macro.try.html), I'm not sure that the `?` operator is 100% equivalent to the `try!` macro, but I think it's good to migrate to the preferred form, and if there's a difference, I imagine we'll find out at compile time.